storm-server/src/main/java/org/apache/storm/container/cgroup/CgroupManager.java

```diff
@@ -305,8 +302,7 @@ private void prefixNumaPinning(List<String> command, String numaId) {
         return newCommand;
     }
 
-    @Override
-    public Set<Long> getRunningPids(String workerId) throws IOException {
+    private Set<Long> getRunningPids(String workerId) throws IOException {
```

Comment:
Does this method (or its variant getAllPids) need to be in the ResourceIsolationInterface?

Reply:
There might be different ways of killing processes for a worker, depending on the implementation, docker, runc. So this method is not general enough to be placed in the interface